scripts/sysman/engine.yml

```diff
@@ -69,11 +69,13 @@ name: $s_engine_stats_t
 members:
     - type: uint64_t
       name: "activeTime"
-      desc: "[out] Monotonic counter for time in microseconds that this resource is actively running workloads."
+      desc: |
+            [out] Monotonic counter where the resource is actively running workloads.
+            Time units are intentionally obfuscated since the activeTime value is only intended for calculating utilization percentage as noted above.
     - type: uint64_t
       name: "timestamp"
       desc: |
-            [out] Monotonic timestamp counter in microseconds when activeTime counter was sampled.
+            [out] Monotonic counter when activeTime counter was sampled.
             This timestamp should only be used to calculate delta time between snapshots of this structure.
```

Comment:
Can we remove "time" from here as well.
I think this entire sentence could be removed, since we cannot calculate time using the timestamp (after this change).

Reply:
you mean from the name of the field? that we can't remove in minor version update since it would break compatibility.

Reply:
> you mean from the name of the field? that we can't remove in minor version update since it would break compatibility.

No in "to calculate delta time betw", we are mentioning that delta time could be calculated.
So either we could have "to calculate delta betw" or may be we could remove the sentence

Reply:
@joshuaranjan fixed